docs/04-operation-guides/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -1,44 +1,123 @@
 ---
-title: Did not receive an event - basic diagnostics
+title: Kyma Eventing - Basic Diagnostics
 ---
 
-## Symptom
+## Symptom(s)
```

Comment:
```suggestion
## Symptom
```
It's a [standardized headline](https://raw.githubusercontent.com/kyma-project/community/main/templates/resources/troubleshooting.md).